api/refresh_token_logic.py

```diff
@@ -0,0 +1,83 @@
+from flask import request
+from user_agents import parse
+import requests
+import hashlib
+import time
+import os
+from typing import Optional
+
+class User_info_service:
```

Comment:
You sure we need a whole another service for this? Why not just put everythin inside our existing UserService? All the info we work on here is user-related. Plus - this service seems like it depends on UserService, since all the login logic happens there